src/components/MessagesList/MessagesGroup/Message/MessagePart/MessageBody.vue

```diff
@@ -39,7 +39,8 @@
 			<Quote v-if="message.parent" :message="message.parent" />
 
 			<!-- Message content / text -->
-			<NcRichText :text="renderedMessage"
+			<NcRichText v-intersection-observer="onIntersectionObserver"
```

Comment:
I would put both directives on a root element. Then it would be the following flow for the component:

![image](https://github.com/nextcloud/spreed/assets/93392545/ae50fdc4-1d2c-4230-bc86-85c82cfca6aa)